bluesky/tests/test_scientific.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+from bluesky.plans import scan
+from bluesky.examples import motor, noisy_det, det
+from bluesky.callbacks.scientific import PeakStats
+
+def test_peak_statistics(fresh_RE):
+    """peak statistics calculation on simple gaussian function
+    """
+    RE = fresh_RE
+    x = 'motor'
+    y = 'det'
+    ps = PeakStats(x, y)
+    RE.subscribe(ps)
+    RE(scan([det], motor, -5, 5, 100))
+
+    assert ps.cen == 0
```

Comment:
maybe `np.isclose(ps.cen, 0, atol=1e-6)` or something similar? Just in case since they're floating point.

Reply:
good catch

Reply:
Might be a good place to use https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.testing.assert_almost_equal.html although `isclose` is fine too.